packages/algob/src/types.ts

```diff
@@ -312,6 +312,7 @@ export interface LsigInfo {
   creator: AccountAddress
   contractAddress: string
   lsig: LogicSig
+  deleted: boolean
```

Comment:
Could you please explain why we add deleted flag in logic signature info ?

Reply:
thanks, removed